modules/ROOT/pages/6.5-release-notes.adoc

```diff
@@ -54,6 +54,10 @@ For information on the **Merge Tags** plugin, see: xref:mergetags.adoc[Merge Tag
 
 The following premium plugin has been announced as reaching its end-of-life:
 
+=== Real-Time Collaboration
+
+{productname}'s xref:rtc-introduction.adoc[Real-time Collaboration (RTC)] plugin was officially deactivated on December 31, 2023, and is no longer available for purchase.
```

Comment:
```suggestion
{productname}'s xref:rtc-introduction.adoc[Real-time Collaboration (RTC)] plugin will be deactivated on December 31, 2023, and is no longer available for purchase.
```

?

Reply:
@TheSpyder thanks for picking up that error 😬